### ISSUE TITLE
Not working with nodenv 1.1.2

### ISSUE THREAD

**Issue by saneef** [nodenv/nodenv-package-rehash#12]
Automatically running `nodenv rehash` with this plugin is not working any more.

Let me know, if there is any log or something which can be used for debugging.

**Comment by saneef** [comment]
Is this someway related to the removal of `package.json` https://github.com/nodenv/nodenv/issues/70 ?

**Comment by jasonkarns** [comment]
Finally able to spend some time on this (sorry for the delay).

I'm running nodenv 1.1.2 (latest from homebrew), and this plugin installed via git clone in the `$(nodenv root)/plugins`.

The npm hooks are still getting installed:

```
$ ls -al ../../versions/12.0.0-nightly20181217b54d4a68e3/lib/node_modules/.hooks/
total 0
drwxr-xr-x  4 jasonkarns  admin   128B Dec 21 16:29 ./
drwxr-xr-x  5 jasonkarns  admin   160B Dec 21 16:29 ../
lrwxr-xr-x  1 jasonkarns  admin    80B Dec 21 16:29 postinstall@ -> /usr/local/var/nodenv/plugins/nodenv-package-rehash/bin/../libexec/nodenv-rehash
lrwxr-xr-x  1 jasonkarns  admin    80B Dec 21 16:29 postuninstall@ -> /usr/local/var/nodenv/plugins/nodenv-package-rehash/bin/../libexec/nodenv-rehash
```

And I threw a debugging echo in the postinstall hook and installed lodash globally and the hook is still being run. Gonna do some more digging. It's possible that npm has changed the structure of the lifecycle env vars that we parse in order to determine if the hook is a "global" install or not.

**Comment by jasonkarns** [comment]
So I can't seem to repro this. It's quite possible the root cause was something unrelated (perhaps the homebrew formula, as @saneef proposed).

Regardless, I'm going to close this for now (since it's been so long); unless anyone is still dealing with this? / @jrolfs, @lucascaro, @cornfeedhobo

**Comment by cornfeedhobo** [comment]
Missed your last message. Yeah, I can't reproduce this ever since I repackaged it for Suse. I'm good here. Thanks for checking in!

**Comment by jasonkarns** [comment]
I now suspect the issue here was the buggy npm that shipped with nodes 8.2.0-10.2.1 #22
